index.html

```diff
@@ -34,105 +34,117 @@ <h2 class="cover__subtitle">LET’S CONNECT</h2>
     <section class="projects" id="projects">
       <section class="card1">
         <section class="card1__area">
-          <div>
-            <img class="card1__image" src="./images/card1.svg" alt="card1 image">
+          <div class="card1__pictures">
```

Comment:
- [ ] [REQUIRED] Kindly, try to make the desktop version of the website similar to the figma design. For example: in the figma design, the text is centerd and the navigation menu is visible. In your design, the navigation menu doesn't show and the text is aligned to the left.

Your Design | Figma Design
------------|--------------
![image](https://user-images.githubusercontent.com/60979926/177664974-33022c73-7d1e-44fb-8004-69cc4e791be4.png) | ![image](https://user-images.githubusercontent.com/60979926/177665606-9a0d4776-2769-4332-9d60-92d455dff214.png)